dune-project

```diff
@@ -46,6 +46,8 @@
  (conflicts
   (bitwuzla-cxx
    (< "0.6.0"))
+  (alt-ergo-lib
+   (>= "2.6.1"))
```

Comment:
It should be "< 2.6.1" (we are adding a conflict on all unsupported version rather than saying which version are valid in depopts above)